io.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2025, Igor Opaniuk <igor.opaniuk@oss.qualcomm.com>
```

Comment:
I'd expect something different here.

Reply:
Updated to the same copyright line currently used in the Linux kernel:

```
Copyright (c) 2025, Qualcomm Innovation Center, Inc. All rights reserved.
```